sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/subquery.scala

```diff
@@ -84,6 +84,13 @@ object ScalarSubquery {
       case _ => false
     }.isDefined
   }
+
+  def hasScalarSubquery(e: Expression): Boolean = {
+    e.find {
```

Comment:
@rxin Thank you for the review. I aligned my code to the existing implementation. But I can replace the method call with your suggestion.